src/main/resources/DefaultTenantConfiguration.json

```diff
@@ -0,0 +1,24 @@
+{
+  "configs": [
+    {
+      "id": "fb0ba4d3-6591-4a2c-99a7-d4009d711b4a",
+      "module": "edge-sip2",
+      "configName": "acsTenantConfig",
+      "code":"fs00001000.conf",
```

Comment:
I don't think this default is used without coding for a fallback plan, specifically for this university (Chalmers) and checkout station, and nothing else. It's there for "real" test sample that we could put on mod-config now. I was thinking of removing it at some point because its value is limited and the test/resource has a similar copy with more bogus values. If we want to use the default/file based config then we should code a fallback path and change the file or its value somewhat.. Another TODO.